cmd/pipeline/deploy.go

```diff
@@ -186,8 +185,14 @@ func getPipelineName() (string, error) {
 	if err != nil {
 		return "", err
 	}
-
-	return filepath.Base(workDir), nil
+	name, err := model.GetValidNameFromGitRepo(workDir)
+	if err != nil {
```

Comment:
if it is not a git repo, we cannot deploy it, right?

Reply:
If you set the right repository and branch you can deploy it like:
`okteto pipeline deploy --repository https://github.com/okteto/movies --branch master`
It will create the pipeline with the name of the folder that you are currently in

Reply:
What I would do is to infer the repository first, and then, use the repo to infer the name.
That way we have consistent naming even if using the `okteto pipeline deploy --repository https://github.com/okteto/movies` syntax